Userland/Libraries/LibWeb/WebAssembly/WebAssembly.cpp

```diff
@@ -450,7 +450,10 @@ JS::Value to_js_value(JS::VM& vm, Wasm::Value& wasm_value)
     case Wasm::ValueType::F32:
         return JS::Value(static_cast<double>(wasm_value.to<float>().value()));
     case Wasm::ValueType::FunctionReference: {
-        auto address = wasm_value.to<Wasm::Reference::Func>().value().address;
+        auto ref_ = wasm_value.to<Wasm::Reference::Func>();
```

Comment:
you'll have to patch .type() too, this won't cut it I'm afraid.

Reply:
I don't think we need to patch `Value::type()`. `Wasm::Reference` contains information about null references on its own.